constants/index.js

```diff
@@ -13,6 +13,7 @@ const BAD_REQUEST_CODE = 400,
   OFFSET = 1,
   RESET_REACTION = 'Reaction removed successfully',
   SUBJECT = 'Welcome to Authors Haven! Verify Your Email',
+  NEW_ARTICLE_SUBJECT = 'New article by follwoed author',
```

Comment:
Seems like you are trying to send a notification here but there's a typo. `'New article by follwoed author'` should be `'New article by followed author'`..Thank me later

Reply:
i will fix this, thanks